app/models/user.rb

```diff
@@ -10,6 +10,7 @@ class User < ActiveRecord::Base
   ADMIN_USERS_INDEX_PAGES = 5
   USERS_INDEX_PAGES = 5
   ADMIN_TIDINGS_INDEX_PAGES = 20
+  TEACHER_METHODICAL_WORKS_INDEX_PAGES = 20
```

Comment:
as for me TEACHER_METHODICAL_WORKS_INDEX_PAGES (namespace/controller name/action) is more clear or it isn't?